packages/next/test/fixtures/00-app-dir-ppr-full/app/page.jsx

```diff
@@ -1,10 +1,14 @@
+import Link from 'next/link'
 import React, { Suspense } from 'react'
 import { Dynamic } from '../components/dynamic'
 
 export default () => {
   return (
-    <Suspense fallback={<Dynamic pathname="/" fallback />}>
-      <Dynamic pathname="/" />
-    </Suspense>
+    <>
+      <Suspense fallback={<Dynamic pathname="/" fallback />}>
+        <Dynamic pathname="/" />
+      </Suspense>
+      <Link href='/cart'>to /cart</Link>
```

Comment:
might be worth running prettier on this app

Reply:
We might wanna do that in separate PR to avoid extra changes in this one since we ignore fixtures in prettier currently.